### ISSUE TITLE
error upon exporting

### ISSUE THREAD

**Issue by pegasusearl** [chartinger/krita-unofficial-spine-export#3]
Krita ver 4.2.7.1
Windows 10

Shown when I pressed "OK" in export window.

```
TypeError
Python 3.6.2: python
Wed Nov 13 16:40:57 2019

A problem occurred in a Python script.  Here is the sequence of
function calls leading up to the error, in the order they occurred.

 C:\Users\USER\AppData\Roaming\krita\pykrita\KritaToSpine\uidocumenttools.py in confirmButton(self=<KritaToSpine.uidocumenttools.UIDocumentTools object>)
  126                 widget.adjust(cloneDoc)
  127                 # Save the json from the clone
  128                 self.spineExport.exportDocument(cloneDoc, self.directoryTextField.text())
  129                 # Clone no longer needed
  130                 cloneDoc.close()
self = <KritaToSpine.uidocumenttools.UIDocumentTools object>
self.spineExport = <KritaToSpine.SpineExport.SpineExport object>
self.spineExport.exportDocument = <bound method SpineExport.exportDocument of <KritaToSpine.SpineExport.SpineExport object>>
cloneDoc = <PyKrita.krita.Document object>
self.directoryTextField = <PyQt5.QtWidgets.QLineEdit object>
self.directoryTextField.text = <built-in method text of QLineEdit object>

 C:\Users\USER\AppData\Roaming\krita\pykrita\KritaToSpine\SpineExport.py in exportDocument(self=<KritaToSpine.SpineExport.SpineExport object>, document=<PyKrita.krita.Document object>, directory='C:/Users/USER/Documents/Pribadi/benteng-takestev-unity/chara')
   41             Krita.instance().setBatchmode(True)
   42             self.document = document
   43             self._export(document.rootNode(), directory)
   44             Krita.instance().setBatchmode(False)
   45             with open('{0}/{1}'.format(directory, 'spine.json'), 'w') as outfile:
self = <KritaToSpine.SpineExport.SpineExport object>
self._export = <bound method SpineExport._export of <KritaToSpine.SpineExport.SpineExport object>>
document = <PyKrita.krita.Document object>
document.rootNode = <built-in method rootNode of Document object>
directory = 'C:/Users/USER/Documents/Pribadi/benteng-takestev-unity/chara'

 C:\Users\USER\AppData\Roaming\krita\pykrita\KritaToSpine\SpineExport.py in _export(self=<KritaToSpine.SpineExport.SpineExport object>, node=<PyKrita.krita.Node object>, directory='C:/Users/USER/Documents/Pribadi/benteng-takestev-unity/chara', bone='root', xOffset=0, yOffset=0, slot=None)
  111             name = self.mergePattern.sub('', child.name()).strip()
  112             layer_file_name = '{0}/{1}.{2}'.format(directory, name, self.fileFormat)
  113             child.save(layer_file_name, 96, 96)
  114 
  115             newSlot = slot
child = <PyKrita.krita.Node object>
child.save = <built-in method save of Node object>
layer_file_name = 'C:/Users/USER/Documents/Pribadi/benteng-takestev-unity/chara/fly2.png'
TypeError: save(self, str, float, float, InfoObject, QRect = QRect()): not enough arguments
    __cause__ = None
    __class__ = <class 'TypeError'>
    __context__ = None
    __delattr__ = <method-wrapper '__delattr__' of TypeError object>
    __dict__ = {}
    __dir__ = <built-in method __dir__ of TypeError object>
    __doc__ = 'Inappropriate argument type.'
    __eq__ = <method-wrapper '__eq__' of TypeError object>
    __format__ = <built-in method __format__ of TypeError object>
    __ge__ = <method-wrapper '__ge__' of TypeError object>
    __getattribute__ = <method-wrapper '__getattribute__' of TypeError object>
    __gt__ = <method-wrapper '__gt__' of TypeError object>
    __hash__ = <method-wrapper '__hash__' of TypeError object>
    __init__ = <method-wrapper '__init__' of TypeError object>
    __init_subclass__ = <built-in method __init_subclass__ of type object>
    __le__ = <method-wrapper '__le__' of TypeError object>
    __lt__ = <method-wrapper '__lt__' of TypeError object>
    __ne__ = <method-wrapper '__ne__' of TypeError object>
    __new__ = <built-in method __new__ of type object>
    __reduce__ = <built-in method __reduce__ of TypeError object>
    __reduce_ex__ = <built-in method __reduce_ex__ of TypeError object>
    __repr__ = <method-wrapper '__repr__' of TypeError object>
    __setattr__ = <method-wrapper '__setattr__' of TypeError object>
    __setstate__ = <built-in method __setstate__ of TypeError object>
    __sizeof__ = <built-in method __sizeof__ of TypeError object>
    __str__ = <method-wrapper '__str__' of TypeError object>
    __subclasshook__ = <built-in method __subclasshook__ of type object>
    __suppress_context__ = False
    __traceback__ = <traceback object>
    args = ('save(self, str, float, float, InfoObject, QRect = QRect()): not enough arguments',)
    with_traceback = <built-in method with_traceback of TypeError object>

The above is a description of an error in a Python program.  Here is
the original traceback:

Traceback (most recent call last):
  File "C:\Users\USER\AppData\Roaming\krita\pykrita\KritaToSpine\uidocumenttools.py", line 128, in confirmButton
    self.spineExport.exportDocument(cloneDoc, self.directoryTextField.text())
  File "C:\Users\USER\AppData\Roaming\krita\pykrita\KritaToSpine\SpineExport.py", line 43, in exportDocument
    self._export(document.rootNode(), directory)
  File "C:\Users\USER\AppData\Roaming\krita\pykrita\KritaToSpine\SpineExport.py", line 113, in _export
    child.save(layer_file_name, 96, 96)
TypeError: save(self, str, float, float, InfoObject, QRect = QRect()): not enough arguments


```

**Comment by chartinger** [comment]
Are you using the current version of this script? It seems to be the same error as in #2

**Comment by pegasusearl** [comment]
Yes I'm pretty sure it's last one. I only installed krita and this script last week with `Tools > Script > Import Python Plugin`.

edit: I used that method because I don't know where `pykrita` folder is at that time.

**Comment by pegasusearl** [comment]
I tried at home with my manjaro linux, installing it with exact same method. It's working fine. But it's abit different. In manjaro, it did not show the same window dialog that appeared in Windows. I just got asked where I want to export then it proceed to export right after I choose folder.

In windows there is a dialog with a form to input where to export and from which document. Then we need to press `Ok`. ... maybe it was... the old version? Will try to reinstall tomorow, since last update was 4 month ago and I barely touch that windows PC in more than 2 month, I thought it was lastest.
